cockatrice/src/sequenceEdit/ui_shortcutstab.h

```diff
@@ -0,0 +1,1597 @@
+    SequenceEdit *Player_aUnattach;
+    QLabel *lbl_Player_aClone;
+    SequenceEdit *Player_aClone;
+    QLabel *lbl_Player_aCreateToken;
+    SequenceEdit *Player_aCreateToken;
+    QLabel *lbl_Player_aCreateAnotherToken;
+    SequenceEdit *Player_aCreateAnotherToken;
+    QLabel *lbl_Player_aSetAnnotation;
+    SequenceEdit *Player_aSetAnnotation;
+    QSpacerItem *verticalSpacer_2;
+    QWidget *tab_3;
+    QGridLayout *gridLayout_20;
+    QGroupBox *groupBox_15;
+    QGridLayout *gridLayout_15;
+    QLabel *lbl_Player_aMoveToBottomLibrary;
+    SequenceEdit *Player_aMoveToBottomLibrary;
+    QLabel *lbl_Player_aMoveToTopLibrary;
+    SequenceEdit *Player_aMoveToTopLibrary;
+    QLabel *lbl_Player_aMoveToGraveyard;
+    SequenceEdit *Player_aMoveToGraveyard;
+    QLabel *lbl_Player_aMoveToExile;
+    SequenceEdit *Player_aMoveToExile;
+    QLabel *lbl_Player_aMoveToHand;
+    SequenceEdit *Player_aMoveToHand;
+    QGroupBox *groupBox_16;
+    QGridLayout *gridLayout_16;
+    QLabel *lbl_Player_aViewGraveyard;
+    SequenceEdit *Player_aViewGraveyard;
+    QLabel *lbl_Player_aViewLibrary;
+    SequenceEdit *Player_aViewLibrary;
+    QLabel *lbl_Player_aViewTopCards;
+    SequenceEdit *Player_aViewTopCards;
+    QLabel *lbl_Player_aViewSideboard;
+    SequenceEdit *Player_aViewSideboard;
+    QLabel *lbl_Player_aViewRfg;
+    SequenceEdit *Player_aViewRfg;
+    QLabel *lbl_GameView_aCloseMostRecentZoneView;
+    SequenceEdit *GameView_aCloseMostRecentZoneView;
+    QGroupBox *groupBox_17;
+    QGridLayout *gridLayout_18;
+    SequenceEdit *DeckViewContainer_loadRemoteButton;
+    SequenceEdit *DeckViewContainer_loadLocalButton;
+    QLabel *lbl_DeckViewContainer_loadRemoteButton;
+    QLabel *lbl_DeckViewContainer_loadLocalButton;
+    QGroupBox *groupBox_18;
+    QGridLayout *gridLayout_19;
+    QLabel *lbl_Player_aDrawArrow;
+    SequenceEdit *Player_aDrawArrow;
+    QLabel *lbl_TabGame_aLeaveGame;
+    SequenceEdit *TabGame_aLeaveGame;
+    QLabel *lbl_TabGame_aRemoveLocalArrows;
+    SequenceEdit *TabGame_aRemoveLocalArrows;
+    QLabel *lbl_TabGame_aConcede;
+    SequenceEdit *TabGame_aConcede;
+    QLabel *lbl_Player_aRollDie;
+    SequenceEdit *Player_aRollDie;
+    QLabel *lbl_TabGame_aRotateViewCW;
+    SequenceEdit *TabGame_aRotateViewCW;
+    QLabel *lbl_Player_aShuffle;
+    SequenceEdit *Player_aShuffle;
+    QLabel *lbl_TabGame_aRotateViewCCW;
+    SequenceEdit *TabGame_aRotateViewCCW;
+    QGroupBox *groupBox_14;
+    QGridLayout *gridLayout_14;
+    QLabel *lbl_Player_aMulligan;
+    SequenceEdit *Player_aMulligan;
+    QLabel *lbl_Player_aDrawCard;
+    SequenceEdit *Player_aDrawCard;
+    QLabel *lbl_Player_aDrawCards;
+    SequenceEdit *Player_aDrawCards;
+    QLabel *lbl_Player_aUndoDraw;
+    SequenceEdit *Player_aUndoDraw;
+    QLabel *lbl_Player_aAlwaysRevealTopCard;
+    SequenceEdit *Player_aAlwaysRevealTopCard;
+    QSpacerItem *verticalSpacer_3;
+    QWidget * tab_4;
+
+    void setupUi(QWidget *shortcutsTab)
```

Comment:
My only concern with this is that it will be very difficult to change the layout in the future as it's all hand done